src/main/java/services/Library.java

```diff
@@ -1,12 +1,14 @@
-package Library;
+package services;
 
-import Data.BookHandler;
+import model.Book;
+import model.Borrower;
+import repositories.BookHandlerImpl;
 
 import java.util.List;
 
 public class Library {
 
-    private BookHandler bookHandler = new BookHandler();
+    private BookHandlerImpl bookHandler = new BookHandlerImpl();
```

Comment:
Keep fields in interfeace types and initialze it in default constructor:  private BookHandler bookHandler;